CHANGES.md

```diff
@@ -8,6 +8,8 @@ releases are available on [Anaconda.org](https://anaconda.org/conda-forge/gettsi
 
 - {gh}`700` Data columns overwrite functions regardless of time unit
   ({ghuser}`lars-reimann`).
+- {gh}`694` Replace `tu` groupings with correct groupings. ({ghuser}`MImmesberger`,
+  \{ghuser} `lars-reimann`, \{ghuser} `hmgaudecker`).
```

Comment:
```suggestion
  {ghuser}`lars-reimann`, {ghuser}`hmgaudecker`).
```

Reply:
Are the pre-commit hooks messing this up? Or was it the extra whitespace?

Reply:
Probably the whitespace together with the hooks.